experimental/dds/tree2/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -3,113 +3,30 @@
  * Licensed under the MIT License.
  */
 import { strict as assert } from "assert";
+import { AsyncGenerator, takeAsync } from "@fluid-internal/stochastic-test-utils";
+import { DDSFuzzModel, DDSFuzzTestState } from "@fluid-internal/test-dds-utils";
 import {
-	AsyncGenerator,
-	makeRandom,
-	performFuzzActionsAsync,
-	SaveInfo,
-} from "@fluid-internal/stochastic-test-utils";
-import { moveToDetachedField, compareUpPaths, rootFieldKeySymbol, UpPath } from "../../../core";
-import { brand } from "../../../util";
+	createDDSFuzzSuite,
+	DDSFuzzHarnessEvents,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluid-internal/test-dds-utils/dist/ddsFuzzHarness";
```

Comment:
you should be able to import these from the main index. if that isn't true, go ahead and add them--these are all intended public APIs.

Reply:
updated index, and import